admin/app/view_models/workarea/admin/user_view_model.rb

```diff
@@ -57,7 +57,14 @@ def payment_profile
 
       def orders
         return [] unless model.email.present?
-        @orders ||= OrderViewModel.wrap(Order.recent(model.id, 50))
+
+        @orders ||= OrderViewModel.wrap(
+          Order
+            .where(email: model.email, :placed_at.exists => true)
```

Comment:
It would be better to reuse the `placed` scope, which will eliminate the need for this `exists` check and the `excludes` below.

Reply:
Note that `Order.recent` does _not_ use the `:placed` scope, we might want to change that at some point in the future.